avm/res/compute/gallery/main.bicep

```diff
@@ -9,27 +9,40 @@ param name string
 @sys.description('Optional. Location for all resources.')
 param location string = resourceGroup().location
 
+@sys.description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+// ============ //
+// Parameters   //
+// ============ //
+
```

Comment:
What's with the parameters above? 😄

Reply:
Again copied from "some template" and thought this had a purpose.

Reply:
They are... also... never mind.